src/libraries/iterableMap.se

```diff
@@ -49,3 +57,6 @@ def contains(key):
 
 def count():
     return self.numberOfItems
+
+def getOffsetByKey(key):
+    return self.itemsMap[key].offset
```

Comment:
Recommend not exposing this.  It is an implementation detail and shouldn't be relied on by users.